test/Pattern/CallbackCacheTest.php

```diff
@@ -11,14 +11,14 @@
 use Laminas\Cache;
 use LaminasTest\Cache\Pattern\TestAsset\FailableCallback;
 use LaminasTest\Cache\Pattern\TestAsset\TestCallbackCache;
+use Laminas\Cache\Exception\InvalidArgumentException;
 
 /**
- * Test function
- * @see LaminasTest\Cache\Pattern\Foo::bar
+ * @see \LaminasTest\Cache\Pattern\Foo::bar
  */
 function bar()
 {
-    return call_user_func_array(__NAMESPACE__ . '\TestAsset\TestCallbackCache::bar', func_get_args());
+    return call_user_func_array([TestCallbackCache::class, 'bar'], func_get_args());
```

Comment:
```suggestion
    $args = func_get_args();
    return TestCallbackCache::bar(...$args);
```

We have the splat operator; let's simplify!